MAINTAINERS_GUIDELINES.md

```diff
@@ -73,4 +73,16 @@ Inactive maintainers are proposed for review by any other [Maintainer](GOVERNANC
 
 An inactive person is defined as someone with less than 10 recorded contributions within the past six months.
 
-Exceptions are allowed for vacation, sick leave, maternity and paternity leave, or planned absences. Moreover, since this method does not consider special situations and does not track all [Maintainers](GOVERNANCE.md#maintainers) duties, other exceptions can be made at the discretion of existing maintainers. In particular, the criteria can be loose and tightened as needed for [*Special*](REPOSITORIES.md#status) repositories and those with very little activity.
\ No newline at end of file
+Exceptions are allowed for vacation, sick leave, maternity and paternity leave, or planned absences. Moreover, since this method does not consider special situations and does not track all [Maintainers](GOVERNANCE.md#maintainers) duties, other exceptions can be made at the discretion of existing maintainers. In particular, the criteria can be loose and tightened as needed for [*Special*](REPOSITORIES.md#status) repositories and those with very little activity.
+
+## Mentoring programs
```

Comment:
just to be clear, I wasn't suggesting putting this in the governance docs.  It's fine to put it there, just don't do it because of me.